storyblok/components.json

```diff
@@ -3501,6 +3501,22 @@
           "type": "image",
           "pos": 9,
           "display_name": "SEO Meta og:image"
+        },
+        "hreflang_en": {
+          "type": "multilink",
+          "link_scope": "en/"
+        },
+        "hreflang_sv": {
+          "type": "multilink",
+          "link_scope": "sv/"
+        },
+        "hreflang_no": {
+          "type": "multilink",
+          "link_scope": "no/"
+        },
+        "Hreflang": {
+          "type": "section",
+          "keys": ["hreflang_en", "hreflang_sv", "hreflang_no"]
```

Comment:
Can we kill of this stuff now maybe? Seems it is not used in the `<head>` anymore? (and all the rests from it)

Reply:
Oops, of course! Thanks for catching that!